TarponSimulator2017/Core/Boat.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.IO;
+using System.Collections;
+using System.Windows;
+
+using Microsoft.Xna.Framework.Input;
+
+namespace Core
+{
+	public class Boat
+	{
+		public Boat ()
+		{
+		}
+
+		public Vector Position
```

Comment:
Use an [auto-implemented property](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties) here. You don't need to manually define a protected variable.

Here is my proposition:

```c#
public Vector Position { get; set; }
// remove: protected Vector _position;
// replace _position by Position in your class
```

It's possible in the future we only want people to get Position and not modify it, so just write:

```c#
public Vector Position { get; private set; }
```

You'll find some examples here: https://github.com/Lesterpig/winnie/blob/master/Winnie/Core/Game.cs#L57